packages/datadog-instrumentations/src/jest.js

```diff
@@ -415,6 +418,20 @@ function cliWrapper (cli, jestVersion) {
       status = 'fail'
       error = new Error(`Failed test suites: ${numFailedTestSuites}. Failed tests: ${numFailedTests}`)
     }
+    let timeoutId
+
+    const flushPromise = new Promise((resolve) => {
```

Comment:
A bit difficult to read and understand the concept here, maybe a little comment like
```suggestion
    // leak the resolve callback to defer it to DC listener
    const flushPromise = new Promise((resolve) => {
```

Reply:
added a comment, thanks